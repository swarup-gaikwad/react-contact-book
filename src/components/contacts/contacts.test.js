import React from "react";
import { shallow } from "enzyme";
import Contacts from "./contacts";

describe("Contacts", () => {
  it("should render correctly", () => {
    const component = shallow(<Contacts />);
  });
  it("should render initial layout", () => {
    // when
    const component = shallow(<Contacts />);
    // then
    expect(component.getElements()).toMatchSnapshot();
  });
});
