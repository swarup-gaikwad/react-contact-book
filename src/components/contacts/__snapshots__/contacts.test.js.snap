// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts should render initial layout 1`] = `
Array [
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-12"
      >
        <h2
          className="font-weight-normal"
        >
          No contact found
        </h2>
      </div>
    </div>
  </div>,
]
`;
